### PR TITLE
Remove app label in phoenix dashboards

### PR DESCRIPTION
**Pull request by fiunchinho**
Towards https://github.com/giantswarm/giantswarm/issues/30932

### Checklist

- [X] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

**Review by QuentinBisson — APPROVED** [review]

**Comment by fiunchinho** [comment]
hey @QuentinBisson [I see that `node` is still being used here](https://github.com/giantswarm/dashboards/blob/main/helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/conntrack.json#L101). What can I use instead of `node`?

Same goes for `role`, [I found it here](https://github.com/giantswarm/dashboards/blob/main/helm/dashboards/charts/public_dashboards/dashboards/shared/public/nodes-overview.json#L81).

**Review by QuentinBisson — APPROVED** [review]
I think it looks good

File: helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/capa-aggregated-error-logs.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
           "type": "prometheus",
           "uid": "${prometheus_datasource}"
         },
-        "definition": "label_values(up{app=\"kubernetes\"},cluster_id)",
+        "definition": "label_values(up{job=\"kubernetes\"},cluster_id)",
```
Comment (marieroque) [inline helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/capa-aggregated-error-logs.json:109]: For `app=kubernetes`, job is equal to `apiserver`, not `kubernetes`.
You can rely on `service=kubernetes`

File: helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/capi-aggregated-error-logs.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
             "uid": "${loki_datasource}"
           },
           "editorMode": "builder",
-          "expr": "{app=~\"cluster-api\"} |~ `(?i)error` |= `$cluster` | logfmt",
+          "expr": "{job=~\"cluster-api\"} |~ `(?i)error` |= `$cluster` | logfmt",
```
Comment (marieroque) [inline helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/capi-aggregated-error-logs.json:55]: For `app=cluster-api`, job is equal to `api-exporter`, not `cluster-api`.
You can rely on `name=cluster-api`

File: helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/capa-aggregated-error-logs.json [hunk 3]
```diff
@@ -52,7 +52,7 @@
             "uid": "${loki_datasource}"
           },
           "editorMode": "builder",
-          "expr": "{app=~\"cluster-api-provider-aws|capa-iam-operator|aws-resolver-rules-operator|irsa-operator\"} |~ `(?i)error` |= `$cluster` | logfmt",
+          "expr": "{job=~\"cluster-api-provider-aws|capa-iam-operator|aws-resolver-rules-operator|irsa-operator\"} |~ `(?i)error` |= `$cluster` | logfmt",
```
Comment (marieroque) [inline helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/capa-aggregated-error-logs.json:55]: I'm not sure this one will work as job seems to be empty

File: helm/dashboards/dashboards/mixin/kube-proxy.json [hunk 4]
```diff
@@ -90,7 +90,7 @@
         {
           "datasource": "$datasource",
           "exemplar": true,
-          "expr": "sum(up{app=\"kube-proxy\"})",
+          "expr": "sum(up{job=\"kube-proxy\"})",
```
[the remark this answers is not in the document]
Comment (marieroque) [reply helm/dashboards/dashboards/mixin/kube-proxy.json:93]: So you can use either `container="kube-proxy"` or `job=~".*kube-proxy.*"` or `pod=~"kube-proxy.*"`

```
{__name__="up", app="kube-proxy", cluster_id="rsz7u", cluster_type="workload_cluster", container="kube-proxy", customer="xxx", installation="viking", instance="10.1.30.29:10249", job="rsz7u-prometheus/kube-proxy-rsz7u/0", namespace="kube-system", node="ip-10-1-30-29.eu-central-1.compute.internal", organization="eiot", pipeline="stable", pod="kube-proxy-jthqq", provider="aws", region="eu-central-1", service_priority="highest"}
```

[the remark this answers is not in the document]
Comment (fiunchinho) [reply helm/dashboards/dashboards/mixin/kube-proxy.json]: Ok. I changed it to use `container`. Please take a look.

**Review by marieroque — APPROVED** [review]

**Pull request merged**